techniques/html/H91.html

```diff
@@ -110,93 +108,94 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
+         <h3>事例 4: チェックボックス</h3>
+         <p>この事例は、<code class="language-html">input</code> 要素の <code class="language-html">type</code> 属性から <code class="language-html">checkbox</code> という役割 (role) を持っている。<code class="language-html">label</code> 要素は、<code class="language-html">input</code> 要素の <code class="language-html">id</code> 属性を参照する <code class="language-html">for</code> 属性を介して <code class="language-html">input</code> 要素に関連付けられる。名前 (name) は、<code class="language-html">label</code> 要素から所得され、この場合は「Cheese」となる。<code class="language-html">checked</code> 属性から、<code class="language-html">checked</code> 又は <code class="language-html">unchecked</code> の状態を取得できる。状態は、利用者がコントロールを操作することで変更できる。</p>
          
 <pre xml:space="preserve"><code class="language-html">&lt;label for="cb-1"&gt;Cheese&lt;/label&gt; 
 &lt;input checked id="cb-1" type="checkbox"&gt;</code></pre>
       </section>
       <section class="example" id="example-5-radio-buttons">
-         <h3>Example 5: Radio Buttons</h3>
-         <p>This example has a role of "radio button" from the <code class="language-html">type</code> attribute on the <code class="language-html">input</code> element. Its name comes from the <code class="language-html">label</code> element. Its state can be <code class="language-html">checked</code> or <code class="language-html">unchecked</code> and comes from the <code class="language-html">checked</code> attribute. The state can be changed by the user's interaction with the control.</p>
+         <h3>事例 5: ラジオボタン</h3>
+         <p>この事例では、<code class="language-html">input</code> 要素の <code class="language-html">type</code> 属性から「ラジオボタン」の役割 (role) が割り当てられている。ボタンの名前 (name) は <code class="language-html">label</code> 要素から取得されている。ボタンの状態は <code class="language-html">checked</code> 属性から取得され、<code class="language-html">checked</code> 又は <code class="language-html">unchecked</code> を選択できる。この状態は、利用者がコントロールを操作することで変更できる。</p>
          
 <pre xml:space="preserve"><code class="language-html">&lt;input checked id="r1" name="color" type="radio"&gt;&lt;label for="r1"&gt;Red&lt;/label&gt;
 &lt;input id="r2" name="color" type="radio"&gt;&lt;label for="r2"&gt;Blue&lt;/label&gt;
 &lt;input id="r3" name="color" type="radio"&gt;&lt;label for="r3"&gt;Green&lt;/label&gt;</code></pre>
       </section>
 			<section class="example" id="example-6-radio-fieldset">
-				<h3>Example 6: Radio Fieldset</h3>
-				<p>The radio <code class="language-html">fieldset</code> has a role of "grouping". The name comes from the <code class="language-html">legend</code> element.</p>
+				<h3>事例 6: ラジオボタンフィールド</h3>
+				<p>ラジオボタン <code class="language-html">fieldset</code> は「グループ化」の役割 (role) を持つ。名前 (name) は <code class="language-html">legend</code> 要素に由来している。</p>
```

Comment:
原文が、他の事例と同じなので (... has a role of "xxx")、日本語訳も他の事例と揃えてみました。また、「The name comes from ...」の訳も、他の事例と揃えてみました。

```suggestion
				<p>ラジオボタン <code class="language-html">fieldset</code> は「グループ化」という役割 (role) を持っている。名前 (name) は <code class="language-html">legend</code> 要素から取得されている。</p>
```